server/index.js

```diff
@@ -12,8 +12,8 @@ const app = express();
 app.use(cors());
 app.use(logger("dev"));
 
-const dirEntry = process.env.DATA_DIR;
-const filePattern = process.env.FILE_PATTERN;
+const dirEntry = process.env.DATA_DIR || "../data";
```

Comment:
Kleiner Hinweis (muss aber nicht umgesetzt werden):

Nutze am besten den [nullush coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).

Die ``||`` zu verenden, wenn etwas nicht gesetzt/spezifiziert wurde ist ein guter Ansatz. Leider nutzt die Doppel-Pipe JavaScripts typische Bool-Konvertierung. Zum Beispiel ``"0"`` und ``0`` usw. werden dann als nicht gesetzt erkannt, auch wenn sie in der Konfig gesetzt wurden. Du kannst ``||`` einfach mit ``??`` austauschen. Das erkennt nur ``null`` und ``undefined`` als false-Fälle an. Ist ein recht neues JS-Feature, ab Node 14 aber verfügbar. Sollte also bei dir locker passen.